static/app/views/alerts/rules/metric/details/metricHistory.tsx

```diff
@@ -20,34 +20,35 @@ type Props = {
   incidents?: Incident[];
 };
 
-function MetricHistory({incidents, activations}: Props) {
+function MetricHistory({incidents}: Props) {
   const organization = useOrganization();
   const sortedActivity = useMemo(() => {
     const filteredIncidents = (incidents ?? []).filter(
       incident => incident.activities?.length
     );
     const activationTriggers: ActivationTriggerActivity[] = [];
-    activations?.forEach(activation => {
-      activationTriggers.push({
-        type: ActivationTrigger.ACTIVATED,
-        activator: activation.activator,
-        conditionType: activation.conditionType,
-        dateCreated: activation.dateCreated,
-      });
-      if (activation.isComplete) {
-        activationTriggers.push({
-          type: ActivationTrigger.FINISHED,
-          activator: activation.activator,
-          conditionType: activation.conditionType,
-          dateCreated: activation.finishedAt,
-        });
-      }
-    });
+    // NOTE: disabling start/finish trigger rows for now
```

Comment:
any idea how long this will be disabled for? (just thinking we could pull it out of git history if it's going to be a while, and clean up this file a little more for now)

Reply:
I'll leave a note.

Just until we determine whether it's useful or not during EA